RyanJacksonHelloWorld/MyApp/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "",
+  "description": "",
```

Comment:
What is the description of your app? I need to update mine as well, but yours should be filled out.